test_scripts/Smoke/HeartBeat/ATF_HeartBeat_no_heartbeat_v2_protocol.lua

```diff
@@ -0,0 +1,99 @@
+--  Requirement summary:
+--  [Services]: SDL must support Heartbeat over protocol v3 or higher
+--
+--  Description:
+--  Check that no heartbeat occurs if App uses v2 protocol version.
```

Comment:
@LitvinenkoIra, will be better to specify heartbeat timeout, instead of heartbeat only.